Jenkinsfile

```diff
@@ -208,20 +214,23 @@ def getDockerImage(Map conf=[:])
         dockerArgs = dockerArgs + " --no-cache "
     }
     echo "Docker Args: ${dockerArgs}"
-    def image = getDockerImageName(prefixpath)
-    //Check if image exists 
-    def retimage
-    try 
-    {
+
+    def image = getDockerImageName(dockerArgs)
+
+    def dockerImage
+    try{
         echo "Pulling down image: ${image}"
-        retimage = docker.image("${image}")
-        retimage.pull()
+        dockerImage = docker.image("${image}")
+        dockerImage.pull()
     }
     catch(Exception ex)
```

Comment:
please add this to avoid catching an Interrupt signal
catch (org.jenkinsci.plugins.workflow.steps.FlowInterruptedException e){
                echo "The job was cancelled or aborted"
                throw e
            }

Reply:
Fixed